testsuite_alp.Dockerfile

```diff
@@ -46,7 +46,7 @@ ARG BUILD_DEPS="\
     tinysparql-dev \
     "
 
-FROM alpine:3.22@sha256:4bcff63911fcb4448bd4fdacec207030997caf25e9bea4045fa6c8c44de311d1 AS build
+FROM alpine:3.22.1@sha256:4bcff63911fcb4448bd4fdacec207030997caf25e9bea4045fa6c8c44de311d1 AS build
```

Comment:
The preferred way to bump container dependencies is through Dependabot!

If you go to https://github.com/Netatalk/netatalk/network/updates you can trigger an update job for Dockerfiles manually. They're scheduled to run automatically at a weekly cadence.

Besides, I think we already use 3.22.1 since "3.22" matches the latest patch version.

All that said, it is not a bad idea to use the patch version tag since this gives better transparency of which version we're actually using. I'm pretty sure Dependabot will keep it updated for us. Can we please update the main [Dockerfile](https://github.com/Netatalk/netatalk/blob/main/Dockerfile) too?

Reply:
Hi. Ok cool, I'll remove this change from the PR and trigger the update that way. 
The reason I updated was because some of the packages we use are no longer available in the repo for 3.22 anymore

Reply:
Hmm, curious.. 3.22 is now succeeding! 3 packages were failing (can't rememeber exactly)
I wonder if it was transient whilst they pulled some CVE'ed ones and was just waiting for mirror replication..
Anyway, I want to try https://github.com/Netatalk/netatalk/network/updates